modules/op-release-notes-1-4.adoc

```diff
@@ -0,0 +1,139 @@
+
+* The controller and webhook runs as a non-root group, and their superfluous capabilities have been removed to make them more secure.
+
+
+// id="cli-new-features-1-4_{context}"]
+// === Pipelines CLI
+
+* You can see streaming logs for task runs that are retried using the `tkn pr logs` command.
+
+* You can use the `--clustertask` option in the `tkn tr delete` command to delete all the task runs associated with a particular cluster task.
+
+
+// [id="triggers-new-features-1-4_{context}"]
+// === Triggers
+
+* Support for using Knative service with the `EventListener` resource is added by introducing a new `customResource` field.
+
+* An error message is displayed when an event payload does not use the JSON format.
+
+* The source control interceptors such as GitLab, BitBucket, and GitHub, now use the new `InterceptorRequest` or `InterceptorResponse` type interface.
+
+* A new CEL function `marshalJSON` is implemented so that you can encode a JSON object or an array to a string.
+
+* An HTTP handler for serving the CEL and the source control core interceptors is added. It packages four core interceptors into a single HTTP server that is deployed in the `tekton-pipelines` namespace. The `EventListener` object forwards events over the HTTP server to the interceptor. Each interceptor is available at a different path. For example, the CEL interceptor is available on the `/cel` path.
+
+
+[id="deprecated-features-1-4_{context}"]
+== Deprecated features
+
+* The `build-gcs` sub-type in the storage `PipelineResource`, and the image `gcs-fetcher`, is not supported.
+
+* In the `taskRun` of cluster tasks, the label `tekton.dev/task` is removed.
+
+* For webhooks, the value `v1beta1` corresponding to the field `admissionReviewVersions` is removed.
+
+* The `creds-init` helper image for building and deploying is removed.
+
+* In the Triggers spec and binding, the deprecated field `template.Name` is removed in favor of `template.Ref`.
+
+* For EventListeners, the fields `PodTemplate` and `ServiceType` are deprecated in favor of `Resource`.
+
+* The deprecated spec style embedded bindings is removed.
+
+* The `Spec` field has been removed from the `TriggerSpecBinding`.
+
+* The event ID representation is changed from a five-character random string to a UUID.
+
+
+[id="known-issues-1-4_{context}"]
+== Known issues
+
+* You can use Pipeline Metrics and Triggers only on OpenShift Pipelines v4.7.6 or later versions.
+
+* On IBM Power Systems, IBM Z, and LinuxONE, the `tkn hub` command is not supported.
+
+* On IBM Power Systems and IBM Z, a Jib Maven cluster task fails.
+
+* When you run Maven and Jib Maven cluster tasks on an IBM Power Systems (ppc64le) cluster, set the `MAVEN_IMAGE` parameter value to `maven:3.6.3-adoptopenjdk-8`.
+
+* When you run Maven and Jib Maven cluster tasks on an IBM Z (s390x) cluster, set the `MAVEN_IMAGE` parameter value to `maven:3.6.3-adoptopenjdk-11`.
```

Comment:
same here add LinuxONE

When you run Maven and Jib Maven cluster tasks on an IBM Z or LinuxONE (s390x) cluster, ....